back/boxtribute_server/graph_ql/types.graphql

```diff
@@ -86,6 +86,7 @@ type Product {
   createdOn: Datetime
   lastModifiedBy: User
   lastModifiedOn: Datetime
+  deleted: Datetime
```

Comment:
Also, I was thinking of just having a boolean `isDeleted` field via a resolver, instead of making the whole Datetime available to the final user. Would that be a better course of action for this PR? What do you think?

Reply:
Done!